tincr/io/library/genCellLibrary.tcl

```diff
@@ -334,7 +438,26 @@ proc createCellLibrary { {part xc7a100t-csg324-3} {filename ""} } {
         puts "Processing: $cname"
 
         puts $fo "    <cell>"
-        puts $fo "      <type>[get_property NAME $libcell]</type>"
+        set cname [get_property NAME $libcell]
+        puts $fo "      <type>$cname</type>"
+
+        # Mark LUT cells
+        if { [string first "LUT" $cname] == 0 } {
+            puts $fo "        <is_lut>"
+            set num [string range $cname 3 100]
```

Comment:
I would change this to:
```
set num_pins [get_property NUM_PINS $libcell] 
set num [expr {$num_pins - 1}]
```

For a LUT lib cell, the pin count is input pins + output pins. Since the LUT has one output pin, the number of input pins is the total number of pins -1. Again, this is probably better than string parsing.